runtime/src/lib.rs

```diff
@@ -343,52 +343,56 @@ cfg_if::cfg_if! {
             fn contains(call: &Call) -> bool {
                 match call {
                     // Allowed calls:
-                    Call::System(_) | Call::Grandpa(_) | Call::Sudo(_) | Call::Timestamp(_) => true,
-
-                    // Prohibited calls:
-                    Call::Balances(_)
-                    | Call::Treasury(_)
-                    | Call::AdvisoryCommittee(_)
+                    Call::AdvisoryCommittee(_)
                     | Call::AdvisoryCommitteeMembership(_)
+                    | Call::Balances(_)
                     | Call::Council(_)
                     | Call::CouncilMembership(_)
-                    | Call::TechnicalCommittee(_)
-                    | Call::TechnicalCommitteeMembership(_)
-                    | Call::MultiSig(_)
+                    | Call::Currency(_)
```

Comment:
We enable `Currency` to allow users to transfers tokens. We have not exposed `orml-tokens` `Call` struct to the runtime, which means one cannot create a transaction that invokes `orml-tokens` to transfer assets. Instead, we have only included `orml-currencies` `Call` struct to encourage transferring any tokens (and if desired also the native currency) using that pallet. Thinking ahead: I am not quite sure if it is a good idea to remove the `Call` struct of `pallet-balances` from the runtime, but this would be a possibility, making `Currency` the only pallet used to manage any kind of funds.